docs/ARCHITECTURE.md

```diff
@@ -0,0 +1,24 @@
+# Architecture
+
+Keptn-Gitea-Provisioner-Service is a microservice with a simple job: When a Keptn project without an upstream is created, it creates a git repo and a token on a (remote) Gitea installation, and return the Git Remote URL, username and token, e.g.:
+
+```
+POST /repository
+  project=foobar
```

Comment:
The request that the provisioner has to process is structured a little bit differently: https://keptn.sh/api/#/repository/addInstance

Reply:
you are right, this was a little too much simplificaiton on my side. fixed it